src/Altinn.Platform/Altinn.Platform.Authorization/Functions/local.settings.json

```diff
@@ -2,9 +2,10 @@
   "IsEncrypted": false,
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "AzureWebJobsSecretStorageType": "files",
     "QueueStorage__queueServiceUri": "https://storageaccountrgdel8df8.queue.core.windows.net",
     "Platform:BridgeApiEndpoint": "http://devenv.altinn.no/sblbridge/authorization/api/",
+    "Platform:AccessTokenIssuer": "https://platform.at21.altinn.cloud/authentication/api/v1/openid/",
```

Comment:
Issuer isn't handled by the AccessTokenGenerator?
A bit unfortunate, as this will create yet another config value that has to be in sync.

Reply:
No, the iss-claim is a required parameter to GenerateAccessToken. Agree that there should be an overload where this doesn't have to be supplied, but for now we have to sort it ourselves (aka in terraform).